docs/io/visualization/montecarlo_packet_visualization.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>Are they referred to as RPackets anywhere else in the documentation? It looks like it's normally just 'real packets'. I would suggest sticking to the same terminology</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yes there in this documentation, it too starts with RPackets- 
https://tardis-sn.github.io/tardis/io/output/rpacket_tracking.html